pyperformance/_benchmark_metadata.py

```diff
@@ -194,7 +194,7 @@ def _resolve(project, tool, filename):
         if target is None:
             target = field
         if field == 'url':
-            repo = project.get('urls', {}).get('repository')
+            _repo = project.get('urls', {}).get('repository')
```

Comment:
Here & elsewhere, better to remove the assignment and/or delete the unused statement than to rename the variable with a leading underscore.